test/externalTests/euler.sh

```diff
@@ -68,6 +68,10 @@ function euler_test
     force_hardhat_unlimited_contract_size "$config_file"
     npm install
 
+    # With ethers.js 5.6.2 many tests for revert messages fail.
+    # TODO: Remove when https://github.com/ethers-io/ethers.js/discussions/2849 is resolved.
+    npm install ethers@5.6.1
```

Comment:
I'm wondering why we use sometimes `npm` and sometimes `yarn`. Is there any special reason for this?

Reply:
I'm trying to stick to the one the project itself uses in its README and `package.json`.

Either one would probably work.